webapp/src/app/components/sidenav/sidenav.component.ts

```diff
@@ -35,22 +35,11 @@ export class SidenavComponent implements OnInit {
 		this.mapLoader.tileMap.subscribe(tilemap => {
 			this.tilemap = tilemap;
 			const currentView = this.globalEvents.currentView;
-			
-			const view = currentView.getValue();
-			currentView.next(EditorView.Layers);
-			if (view !== EditorView.Layers) {
-				currentView.next(view);
-			}
+			currentView.next(currentView.value); //Update select mode
 		});
 		this.globalEvents.currentView.subscribe(view => {
-			this.currentView = view;
-			switch (view) {
-				case EditorView.Layers:
-					this.activeTab = 0;
-					break;
-				case EditorView.Entities:
-					this.activeTab = 1;
-					break;
+			if (view !== undefined && this.activeTab != view) {
+				this.activeTab = view;
```

Comment:
Missed this the first time. `activeTab` is a number and view is not, should be changed to match.

Reply:
Fixed